jobs/advanced-analysis/advanced-analysis-worker/terraform/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.7.0"
+    }
+
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.21.1"
+    }
+  }
+
+  required_version = "1.5.2"
+
+  backend "s3" {
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = var.region
+}
+
+variable "region" {
+  type    = string
+  default = "us-east-2"
+}
+
+resource "aws_ecr_repository" "advanced_analysis_worker_ecr" {
+  name                 = "advanced-analysis-worker"
+  image_tag_mutability = "MUTABLE"
```

Comment:
we might want to make this `IMMUTABLE` I saw the other ecr repo configs have `MUTABLE` but I think that might have been an oversight when making those repos